python/pyarrow/tests/test_feather.py

```diff
@@ -805,6 +805,16 @@ def test_read_column_duplicated_in_file(tempdir):
         read_table(path, columns=['a', 'b'])
 
 
+def test_read_column_wrong_column_type(tempdir, version):
+    # duplicated columns in the column selection
```

Comment:
The comment is a leftover and can be removed. As for the test name, we might change that to reflect the test? For example `test_read_column_with_generator`.

Reply:
yep, sorry!